### PR TITLE
add-lambda-test-util

### PR DESCRIPTION
**Pull request by jgilbert01**

### CODE REVIEW

File: src/utils/lambda-test.js [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+import { debug as d } from './print';
+
+export default ({
+  functionName,
+  endpoint = 'http://localhost:3002',
+  timeout = 1000,
+  debug = d('test'),
+
+}) => {
+  const lambda = new LambdaClient({
```
Comment (petermyers) [inline src/utils/lambda-test.js:15]: Did you intentionally not want to use the lambda connector here?

Comment (petermyers) [reply src/utils/lambda-test.js:15]: Disregard, the lambda connector doesn't provide a way to override with a custom endpoint.

Follow up question, should it?

Comment (jgilbert01) [reply src/utils/lambda-test.js:15]: - good question
- no, i don't think so
- these r totally different use cases so i want to keep them decoupled

File: src/connectors/index.js [hunk 2]
```diff
@@ -1,9 +1,9 @@
...
+// export { default as CloudWatchConnector } from './cloudwatch';
 export { default as DynamoDBConnector } from './dynamodb';
 export { default as EventBridgeConnector } from './eventbridge';
-export { default as FirehoseConnector } from './firehose';
-export { default as KinesisConnector } from './kinesis';
-export { default as LambdaConnector } from './lambda';
+// export { default as FirehoseConnector } from './firehose';
+// export { default as KinesisConnector } from './kinesis';
+// export { default as LambdaConnector } from './lambda';
 export { default as SecretsMgrConnector } from './secretsmgr';
```
Comment (jgilbert01) [inline src/connectors/index.js:7]: - reduce the number of dependencies that must be included
- but keep the most common for easier use

File: src/utils/lambda-test.js [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+    .send(new InvokeCommand({
+      FunctionName: functionName,
+      InvocationType: 'RequestResponse',
+      Payload: JSON.stringify(event),
+    }))
+    .then((resp) => ({
+      ...resp,
+      Payload: JSON.parse(Buffer.from(resp.Payload)),
+    }));
+};
```
Comment (jgilbert01) [inline src/utils/lambda-test.js:42]: provides support for serverless-offline 13.x

**Pull request merged**